libmscore/accidental.cpp

```diff
@@ -369,7 +439,7 @@ void Accidental::draw(QPainter* painter) const
             return;
       painter->setPen(curColor());
       for (const SymElement& e : el)
-            score()->scoreFont()->draw(e.sym, painter, magS(), QPointF(e.x, 0.0));
+            score()->scoreFont()->draw(e.sym, painter, magS(), QPointF(e.x, e.y));
```

Comment:
Why? Looks like a serious change to the logic of drawing accidentals.

Reply:
I don't recall, but omitting the y coordinates seems just plain wrong

Reply:
I believe it is due to the brackets, see https://github.com/musescore/MuseScore/pull/6182/files#diff-ecd72d8d5148bd7cab37cef6908b7d83R379 and https://github.com/musescore/MuseScore/pull/6182/files#diff-ecd72d8d5148bd7cab37cef6908b7d83R386